tox.ini

```diff
@@ -1,29 +1,22 @@
 [tox]
-envlist = coverage-clean, py27, py33, py34, py35, pypy, coverage-report
+envlist = flake8, py27, py34, py35, py36, py37, pypy, pypy3, coverage
 
 [testenv]
+usedevelop = true
 commands =
-    coverage run setup.py ftest
-setenv =
-  COVERAGE_FILE=.coverage.{envname}
+    zope-testrunner --test-path=src []
+extras = test
+
+[testenv:coverage]
+basepython = python3.7
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage report --fail-under=80
 deps =
-    .[test]
     coverage
```

Comment:
(This was surprising to me until I realized it was inheriting `extras = test` from the main `[testenv]`.)